### PR TITLE
Add dyn-asp bib entries

### PR DESCRIPTION
**Pull request by pobermei**

### CODE REVIEW

File: krr.bib [hunk 1]
```diff
@@ -20011,3 +20011,28 @@ @InProceedings{zlapop90
   editor =	 "K. Konolige",
   pages =	 "259-271"
 }
+
+@Article{shstfest15a,
+  author =       "G. Sharon and R. Stern and A. Felner and N. Sturtevant",
+  title =        "Conflict-based search for optimal multi-agent pathfinding",
+  journal =      aij,
+  year =         2015,
+  volume =       219,
```
Comment (tortinator) [inline krr.bib]: Issue?

File: procs.bib [hunk 2]
```diff
@@ -1605,6 +1605,13 @@ @Proceedings{iclp90
   publisher =	 "MIT Press"
 }
 
+@Proceedings{icra86,
+  title =	 "Proceedings of the IEEE International Conference on Robotics and Automation (ICRA'86)",
+  year =	 1986,
+  booktitle =	 "Proceedings of the IEEE International Conference on Robotics and Automation (ICRA'86)",
+  publisher =	 "IEEE"
```
Comment (tortinator) [inline procs.bib]: Abbreviation? IEEE is not the publisher but the organization

File: krr.bib (hunk 1 above)
Comment (pobermei) [reply krr.bib]: There is no issue field for aij

File: procs.bib (hunk 2 above)
Comment (pobermei) [reply procs.bib]: Fixed by 88c6f6b

**Pull request merged**